pytest_tap/tests/test_plugin.py

```diff
@@ -45,11 +49,13 @@ def test_stream(testdir, sample_test_file):
 
     result.stdout.fnmatch_lines(
         [
+            "1..6",
             "ok 1 test_stream.py::test_ok",
             "not ok 2 test_stream.py::test_not_ok",
-            "ok 3 test_stream.py::test_skipped # SKIP some reason",
-            "ok 4 test_stream.py::test_broken # TODO expected failure: a reason",
-            "1..4",
+            "ok 3 test_stream.py::test_params[foo]",
```

Comment:
Cool, this is great! Thanks for adding this to the test suite. I think it adds clarity to what will happen with parameterized tests.